cypress/e2e/mfa/MFA.spec.ts

```diff
@@ -8,10 +8,15 @@ describe("MFA set-up", () => {
     cy.signIn();
   });
   it("should render the Choose MFA page", () => {
-    cy.get('[data-cy="MFA set-up"]').should("exist").click();
+    cy.get('[data-cy="MFA"]').first().click();
+    cy.get('[data-cy="homeLink"]').should("exist");
+    cy.get(".nhsuk-fieldset__heading")
+      .should("exist")
+      .should("include.text", "MFA (Multi-Factor Authentication) set-up");
+
     cy.get("[data-cy=mfaAlreadyWarning]").should("exist");
     cy.get("[data-cy=mfaAlreadyText]")
-      .should("exist")
+      .should("exist")  
```

Comment:
Random bit of whitespace if you feel like cleaning it up

Reply:
Hmm prettier usually catches those on save. Cheers!